sdk/metric/aggregator/minmaxsumcount/mmsc.go

```diff
@@ -28,14 +28,13 @@ type (
 	// Aggregator aggregates events that form a distribution,
 	// keeping only the min, max, sum, and count.
 	Aggregator struct {
-		lock       sync.Mutex
-		current    state
-		checkpoint state
-		kind       metric.NumberKind
+		lock sync.Mutex
+		kind metric.NumberKind
+		state
 	}
 
 	state struct {
-		count metric.Number
+		count int64
```

Comment:
This should be okay given the static bit-width, but might be worth moving to the end of the struct to keep with our alignment standards for struct fields that need 64-bit alignment going at the top.

Reply:
Done.